src/modules/LanItems.rb

```diff
@@ -2312,8 +2198,20 @@ def enableCurrentEditButton
       end
     end
 
-    def createS390Device
-      Builtins.y2milestone("creating device s390 network device")
+    # Creates eth emulation for s390 devices
+    #
+    # @param [Hash] an s390 device description as obtained from AY profile
+    def createS390Device(rule)
```

Comment:
It seems method `createS390Device` is not covered with unit tests. Is it tested in another way?
I see it was mostly copied from another file, but mistakes can happen even with copying.